src/acl/Acl.cc

```diff
@@ -25,6 +25,7 @@
 #include "SquidConfig.h"
 
 #include <algorithm>
+#include <list>
```

Comment:
This new `#include` looks misplaced or unnecessary because the new code that uses `std::list` is in the _header_ file.

Reply:
Fair enough. Removing from here and putting in the header. It's probably indirectly included but it'll be more reliable this way